css/store.css

```diff
@@ -0,0 +1,119 @@
+:root {
+  --form-colour: #828282;
+}
+
+.header {
+  background-color: #f7f7f7;
+}
+
+.main {
+  display: flex;
+  justify-content: space-between; /* Divides the main page into two */
+  margin-bottom: 0;
+  width: 100%;
+}
+
+.form {
+  margin: 0 10%;
+  color: var(--form-colour);
+  font-size: 25px;
+}
+
+label {
+  margin-bottom: 3%;
+}
+
+input[type="text"],
+#city {
+  border: 2px solid var(--form-colour);
+  border-radius: 5px;
+  padding: 10px 5px;
+}
+
+.form-title {
+  font-size: 50px;
+  color: #f15b2a;
+  font-weight: 500;
+}
+
+.form-element {
+  /* These styles are created so that the label and input remain together */
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-right: 5%;
+  margin-bottom: 5%;
+  width: 100%;
+}
+
+.form-names {
+  /* Name section of the form */
+  display: flex;
+}
+
+.form-addresses {
+  /* Address section of the form */
+  display: flex;
+  flex-direction: column;
+}
+
+.address-input {
+  width: 84%; /* Controls the width of the address input element */
+}
+
+.form-location {
+  /* Location section of the form */
+  display: flex;
+}
+
+.city {
+  width: 100%;
+  background-color: #fff;
+}
+
+.postcode {
+  width: 75%;
+}
+
+.postcode-container,
+.space-grey {
+  margin-left: 2%; /* Shared margins */
+}
+
+.form-colour-text {
+  /* Order colour selector text */
+  margin-bottom: 2%;
+}
+
+.form-colour {
+  /* Colour section of the form */
+  margin-bottom: 10%;
+}
+
+.radio {
+  margin-right: 2%;
+}
+
+.check {
+  margin-bottom: 5%;
+}
+
+.order-button {
+  color: #fff;
+  background-color: #f15b2a;
+  border-radius: 2px;
+  border: none;
+  padding: 15px 35px;
+}
+
+.terms {
+  color: #f15b2a;
+}
+
+.image {
+  background-image: url("../img/store-image_by-andrew-neel-unsplash.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 0 25% 60%;
+  margin-bottom: -8px;
+}
```

Comment:
Your store page is wider than the screen allows, which is why it scrolls horizontally. This is because you have not set the width for each half. The form should be 50% and so should the image. 

Check your `.form` class on line 17 - your margin may also be an issue as it will add to the 50%!